groups/sig-release/groups.yaml

```diff
@@ -403,34 +374,19 @@ groups:
       - k8s@auggie.dev
       - saschagrunert@gmail.com
     managers:
+      - fsmunoz@gmail.com # v1.34 EA
+      - jackhammervyom@gmail.com # v1.34 Release Lead
```

Comment:
@katcosgrove I added myself as a manager of the group, I don't know if I require that access (Nina was a member last cycle), let me know if I don't need it (I added myself based on the pattern), and I'll move myself to the `members` list.

Reply:
Update: The current access is correct, it would be discussed later if we should remove EA and Lead.